### PR TITLE
2.x: Rename Flowable Base Interface Types for consistency

### PR DESCRIPTION
**Pull request by vanniktech**
Addresses #4044

### CODE REVIEW

File: src/main/java/io/reactivex/FlowableOperator.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package io.reactivex;
```
Comment (JakeWharton) [inline src/main/java/io/reactivex/FlowableOperator.java:14]: license header

File: src/main/java/io/reactivex/FlowableTransformer.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package io.reactivex;
```
Comment (JakeWharton) [inline src/main/java/io/reactivex/FlowableTransformer.java:14]: license header

**Comment by JakeWharton** [comment]
lgtm :+1:

File: src/main/java/io/reactivex/FlowableOperator.java (hunk 1 above)
Comment (vanniktech) [inline src/main/java/io/reactivex/FlowableOperator.java:14]: yup just noticed it too when I pushed it. added it

**Comment by akarnokd** [comment]
:+1:

**Pull request merged**